instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+1.  **Welcoming to the community.**
+    1.  Once the trainee has passed the demonstration lesson,
+        we generate a PDF certificate for the trainee by running a command-line script in
+        [the certification repository][cert-repo]
+	and send it to her along with instructions for the items below.
+    1.  The instructions sent to the new instructor tell her to:
+        1.  join the `discuss` and `instructors` lists
+	1.  fill in a form on AMY to tell us where she is, what she's comfortable teaching, etc., and
+	1.  send us a short biography and photo to add to [our website][team-page].
+    1.  We manually add the biography and photo to the website
+        (since they almost always require reformatting).
+
+## Resources
+
+1.  [AMY][amy] is the web application we use to track instructors and workshops.
+    Most of it is only accessible to administrators,
+    but it does have a form where instructors can supply or update information about themselves.
+1.  There's a web form on AMY that people can use to apply for instructor training
+    (currently disabled).
+1.  We keep a list of trainees in a Google Docs spreadsheet with the following fields,
+    which we upate manually:
+    1.  name
+    1.  contact email
+    1.  which training course they took part in
+    1.  the name of the Software Carpentry lesson they're concentrating on (if any)
+    1.  the name of the Data Carpentry lesson they're concentrating on (if any)
+    1.  the URL of their Software Carpentry exercise submission (if any)
+    1.  whether or not they have submitted something for Data Carpentry
+    1.  the date and moderator of the discussion session they have taken part in
+        *   we currently store both values in one column
+	*   we do not record both a Software Carpentry and a Data Carpentry discussion session (though we should)
+    1.  the date on which they passed their Software Carpentry lesson demo (if any)
+	*   this date becomes the award date of the PDF certificate
+        *   we do not currently record the name of the person who was their instructor, or the name of the person who did their final test
+    1.  the date on which they passed their Data Carpentry lesson demo (if any)
+        *   the same caveats apply
+1.  Mentors offer lesson discussion sessions,
+    and trainees sign up for them,
+    on an Etherpad.
+1.  Trainers offer checkout sessions on a second Etherpad.
+    Trainees sign up for sessions on that Etherpad,
+    which we also presently use for taking notes during their sessions.
+1.  Discussion and checkout sessions are marked in Software Carpentry's community calendar
+    (a Google calendar).
+
+## Issues
+
+1.  As of Jan 2016,
+    the form for individuals to apply to take part in training has been disabled
+    while we try to clear our backlog (and come up with a better process).
+1.  We used a Google web form and a spreadsheet for group applications and sorted manually.
```

Comment:
@chrismedrela - thanks for your feedback! To your points:

 - search for: instructors with a badge within D miles of [airport code] who have taught < N workshops in the last M months who are comfortable teaching topic X <--- this is the kind of question I would regularly ask when trying to simultaneously staff a workshop and get new or lapsed instructors participating.

 - last paragraph: you're right that the broader connections have nothing to do with what people are discussing in discussion sessions; rather, they're one scheme the steering committee is kicking around for mentorship and leadership to increase the numbers of people who actually bother to go through the discussion and 5-minute lesson (currently only about 2/3). The idea is that if there's a whole lot of people from Society X, we might try and deputize a representative from that group to keep everyone else moving through the process and on board. Therefore, given a new instructor trainee who hasn't finished their checkouts yet, I'd like to know who their mentor is so I can ping them and see if there's a way we can get their cohort moving again. But, as I muse in the wall of text above, it's not 100% thought out yet how we want to make these groupings - more feedback from the rest of the team required.

Thanks for all your hard work, let me know any way I can support you!